docs/reference/providers/faas-providers.md

```diff
@@ -0,0 +1,25 @@
+# `faas-provider` Interface
+The `faas-provider` provides the CRUD API for Functions as well as the invoke capabilities.
```

Comment:
Can you add the diagram from the repo please?

Reply:
Added, here is a preview

![image](https://user-images.githubusercontent.com/891889/71675806-ec313b00-2d7e-11ea-9345-205e852fd46f.png)